### DISCUSSION TITLE
Need review regarding my readme.md

### DISCUSSION THREAD

**Discussion by sarfraz-web** [community/community#124422] [Other Feature Feedback, Questions, & Ideas]
Hey fellow gitians i wanted to ask u all does my readme looks professional or not

**Comment by karthickkumarsm** [comment]
HI sarfaraz! your github profile looks good and professional but it needs some changes.
1.The logo of Kafka and photoshop not clearly visible
2.The profile views and the github skill should be below the introduction part
I used the below website to create my profile readme file:https://gprm.itsvg.in/
Have a look and make it great brother!
@sarfraz20020707

**Comment by ap0calypse8** [comment]
Your profile looks absolutely amazing but try adding a better profile picture that suits the github environment, also try achieving github badges and showcase them on your profile.

To make your GitHub account appear more professional, here are some steps you can take:

Profile Picture: Choose a professional-looking profile picture. It could be a clear headshot or a professional logo if you're representing an organization.

Bio: Write a concise and informative bio that highlights your expertise, interests, and contributions. Keep it professional but also let your personality shine through.

Repositories: Keep your repositories well-organized and properly named. Use clear and descriptive names for your projects, and make sure they have informative README files explaining what the project is about, how to use it, and any other relevant information.

Contributions: Regularly contribute to open-source projects or create your own projects. By looking at your contributions it seemed you are not much active which is not much professional.

Activity: Stay active on GitHub by regularly pushing code, opening and participating in discussions, and reviewing pull requests. This demonstrates your commitment to your projects and the community.

Follow Others: Follow other users and organizations whose work you admire or want to collaborate with. This helps you stay connected and engaged with the GitHub community.

Skills and Technologies: Your skills that you potray in the profile look a bit too much , feels like you dont have a specialisation in any tech stack, redo that part.

Clean up Old Repositories: If you have old or abandoned repositories that no longer reflect your current skills or interests, consider archiving or deleting them to keep your profile focused.

Remember, professionalism on GitHub is not just about appearances but also about your contributions and interactions.

if you liked my answer please upvote it.

**Comment by adarsh-site** [comment]
hey @sarfraz20020707 , your profile is good. But you can further improve it by choosing other set of icons and formatting markdown. Also make sure to pin some of your top repo. You can check mine as a reference.

**Comment by shrawank039** [comment]
looks good but not that professional. 
there are multiple broken images, fix that first.
keep personal info in one place and all the stats in the same place. currently, you are using random alignments.
keep the content low where you don't have that many stats to show. just keep that for the future. 
don't focus that much on these things. by time you will get what to show and what to write here.

**Comment by stablestream** [comment]
I love your profile, but the icons look weird in dark mode.
